<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSlotMag</class>
 <widget class="QWidget" name="WSlotMag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="c_slot_type">
       <item>
        <property name="text">
         <string>Slot Type 50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 51</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 52 R</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 53</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 54</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slot Type 56</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="w_slot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>750</width>
       <height>450</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
